.gitignore

```diff
@@ -33,6 +33,7 @@ nbproject
 .sass-cache
 *.map
 _site
+_tmp_preview_config.yml
```

Comment:
Yes, I had to create a temporary configuration file to get the preview to work. Not ideal, but oh well.